source/common/res/features/budget-daily-spending/main.js

```diff
@@ -0,0 +1,127 @@
+          if (ynabToolKit.dailySpending.budgetView === null) {
+            try {
+              ynabToolKit.dailySpending.budgetView = ynab.YNABSharedLib.getBudgetViewModel_AllBudgetMonthsViewModel()._result;
+            } catch (e) {
+              return;
+            }
+          }
+          var categories = [];
+          var masterCats = ynabToolKit.dailySpending.budgetView
+          .categoriesViewModel.masterCategoriesCollection._internalDataArray;
+          var masterCategories = [];
+
+          masterCats.forEach(function (c) {
+            // Filter out "special" categories
+            if (c.internalName === null) {
+              masterCategories.push(c.entityId);
+            }
+          });
+
+          masterCategories.forEach(function (c) {
+            var accounts = ynabToolKit.dailySpending.budgetView
+              .categoriesViewModel.subCategoriesCollection
+              .findItemsByMasterCategoryId(c);
+
+            Array.prototype.push.apply(categories, accounts);
+          });
+
+          return categories;
+        },
+
+        updateDailySpending(f) {
+          var amount = ynabToolKit.dailySpending.getBalanceAmount(f);
+          var date = new Date();
+          var remainingDays = new Date(date.getFullYear(), date.getMonth() + 1, 0).getDate() - date.getDate();
+          var dailySpendingAmount = amount / remainingDays;
+          if (dailySpendingAmount < 0) {
+            dailySpendingAmount = 0;
+          }
+          var fdailySpending = ynabToolKit.shared.formatCurrency(dailySpendingAmount * 1000);
+          var classDt = 'positive';
+          var classSpan = 'positive';
+          if (dailySpendingAmount === 0) {
+            classSpan = 'zero';
+          }
+          if (dailySpendingAmount === 0) {
+            classDt = '';
+          }
+          var available = $('<dl>', { class: 'inspector-overview-available' })
+          .append($('<dt>', { class: classDt })
+                  .append('Daily Spending')
+                  )
+          .append($('<dd>', { class: 'user-data' })
+                  .append($('<span>', { class: 'user-data currency ' + classSpan })
+                          .append(fdailySpending.toString()))
+                  );
+          $('.budget-inspector-category-overview .inspector').append($('<hr>', { class: 'clearfix' }));
+          $('.budget-inspector-category-overview .inspector').append(available);
+        },
+
+        getInspectorName() {
+          return $('.inspector-category-name.user-data').text().trim();
+        },
+
+        getBalanceAmount(f) {
+          var currentMonth = moment(ynabToolKit.shared.parseSelectedMonth())
+            .format('YYYY-MM');
+          var monthlyBudget = ynabToolKit.dailySpending.budgetView
+            .monthlySubCategoryBudgetCalculationsCollection
+            .findItemByEntityId('mcbc/' + currentMonth + '/' + f.entityId);
+
+          return (monthlyBudget.balance * 0.001);
```

Comment:
`ynabToolKit.shared.formatCurrency` should do this for you and give you and provide currency symbols. If you don't want/need the currency symbol then just use `ynab.normalizeCurrency`